### PR TITLE
New functions and bug fix

### PR DESCRIPTION
**Pull request by JetSetIlly**
Changes for issues #75, #77 and #78

### CODE REVIEW

File: imgui.go [hunk 1]
```diff
@@ -1222,3 +1227,8 @@ func MouseCursor() int {
 func SetMouseCursor(cursor int) {
 	C.iggSetMouseCursor(C.int(cursor))
 }
+
+// SetKeyboardFocusHere gives keyboard focus to next item
+func SetKeyboardFocusHere(offset int) {
```
Comment (dertseha) [inline imgui.go]: `offset` is a default parameter. Please add make this function to be `SetKeyboardFocusHereV` and add a second function `SetKeyboardFocus()` which calls the verbose one with `0` as argument.

**Comment by dertseha** [comment]
Great - almost - one small change for the function with the default parameter please.

**Comment by JetSetIlly** [comment]
Pull request updated.

**Comment by dertseha** [comment]
Great, thank you for the contribution!

**Pull request merged**